Project.toml

```
@@ -1,7 +1,7 @@
 name = "TimeZones"
 uuid = "f269a46b-ccf7-5d73-abea-4c690281aa53"
 authors = ["Curtis Vogt <curtis.vogt@gmail.com>"]
-version = "1.9.0"
+version = "1.10.0"
```

All version changes for this repo are handled outside of PRs
```suggestion
version = "1.9.0"
```